bin/pca-import

```diff
@@ -773,6 +779,59 @@ def import_template_file(template_filename, template_id=None, force=False):
     )
     return True
 
+def user_report_already_exists(customer_id, assessment_id, campaign_id):
+    try:
+        UserReportDoc.objects.raw({"customer": customer_id,
+                                 "assessment": assessment_id,
+                                 "campaign": campaign_id})
+        return True
+    except DoesNotExist:
+        return False
+
+def import_user_report(request, source, force=False):
+    user_report_id = None
+    if not customer_already_exists(request["customer"]):
+        print(
+            "Cannot import, customer_id does not exist: {}".format(
+                request["customer"]
+            )
+        )
+        return False
+    if not assessment_already_exists(request["assessment"]):
+        print(
+            "Cannot import, assessment_id does not exist: {}".format(
+                request["assessment"]
+            )
+        )
+        return False
+    if not campaign_already_exists(request["campaign"]):
+        print("Cannot import, campaign_id does not exist: {}".format(
+            request["campaign"]
+            )
+        )
+        return False
+
+    user_report = UserReportDoc.find_by_customer_assessment_campaign(request["customer"],
+                                                                     request["assessment"],
+                                                                     request["campaign"])
+    if user_report:
+        if force:
+            user_report_id = user_report._id
+        else:
+            print(
+                "Cannot import, user_report already exist - Customer: {} Assessment: {} Campaign: {}".format(
```

Comment:
Fix grammar and match other error messages:
```suggestion
                "Cannot import, user_report already exists - customer_id: {} assessment_id: {} campaign_id: {}".format(
```

Reply:
Suggestion was added in 86972531a8facb739a161eecba3a8c1e546aca54